loopy/statistics.py

```diff
@@ -852,49 +867,49 @@ def map_call(self, expr):
             return subst_into_to_count_map(
                     self.param_space,
                     sub_result, subst_dict) \
-                    + self.rec(expr.parameters)
+                    + self.rec(expr.parameters, *args)
 
         else:
             raise NotImplementedError()
 
-    def map_call_with_kwargs(self, expr):
+    def map_call_with_kwargs(self, expr, *args):
         # See https://github.com/inducer/loopy/pull/323
         raise NotImplementedError
 
-    def map_sum(self, expr):
+    def map_sum(self, expr, tags=None):
         if expr.children:
-            return sum(self.rec(child) for child in expr.children)
+            return sum(self.rec(child, tags) for child in expr.children)
         else:
             return self.new_zero_poly_map()
 
     map_product = map_sum
 
-    def map_comparison(self, expr):
-        return self.rec(expr.left)+self.rec(expr.right)
+    def map_comparison(self, expr, *args):
```

Comment:
Why have `*args` instead of `tags` here?

Reply:
These are hopefully all fixed in [df7ed9b](https://github.com/inducer/loopy/pull/688/commits/df7ed9b5dee8fadf1c984a22985edfce7ecbf454)